integrations/Satismeter/browser.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-var */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+import { LOAD_ORIGIN } from "../ScriptLoader";
+
+class Satismeter {
+  constructor(config, analytics) {
+    logger.debug("in satismetwr const");
+    if (analytics.logLevel) {
+      logger.setLogLevel(analytics.logLevel);
+    }
+    this.name = NAME;
+    this.writeKey = config.writeKey;
+  }
+
+  init() {
+    logger.debug("===In init Satismeter===");
+    (function () {
+      var script = document.createElement("script");
+      var parent = document.getElementsByTagName("script")[0].parentNode;
+      window.satismeter =
+        window.satismeter ||
+        function () {
+          (window.satismeter.q = window.satismeter.q || []).push(arguments);
+        };
+      window.satismeter.l = 1 * new Date();
+      script.async = 1;
+      script.src = "https://app.satismeter.com/js";
+      script.setAttribute("data-loader", LOAD_ORIGIN);
+      parent.appendChild(script);
+    })();
```

Comment:
We should use the `scriptLoader` function here.

Reply:
I don’t think we can as scriptLoader is mainly for those which have an link for their script whereas in this destination's case we are not getting any link its a raw script which we have to run

Reply:
The script is too general to be used and on trying it as well it didn't work. They are using some internal Functioning due which ScriptLoader can not work.